lib/babe/build.go

```diff
@@ -59,30 +59,27 @@ type BlockBuilder struct {
 	keypair               *sr25519.Keypair
 	transactionState      TransactionState
 	blockState            BlockState
-	proof                 *VrfOutputAndProof
 	currentAuthorityIndex uint32
+	preRuntimeDigest      *types.PreRuntimeDigest
 }
 
 // NewBlockBuilder creates a new block builder.
 func NewBlockBuilder(kp *sr25519.Keypair, ts TransactionState,
-	bs BlockState, proof *VrfOutputAndProof,
-	authidx uint32) (*BlockBuilder, error) {
+	bs BlockState,
+	authidx uint32, preRuntimeDigest *types.PreRuntimeDigest) (*BlockBuilder, error) {
```

Comment:
nit equal line lenghts are nice

```suggestion
	bs BlockState, authidx uint32, 
	preRuntimeDigest *types.PreRuntimeDigest) (
	*BlockBuilder, error) {
```